precache/u1/app.js

```diff
@@ -0,0 +1,137 @@
+// Import libraries
+const express = require('express')
+
+// Setup app
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+// localhost:3000
+
+app.get('/', (req, res) => {
+  res.send('Better world')
+})
+
+// localhost:3000/html
+
+app.get('/html', (req, res) => {
+  res.send(`
+      <html><body>
+        <h1>Technologies</h1>
+        <ul>
+          <li>JavaScript</li>
+          <li>Node</li>
+          <li>Express</li>
+        </ul>
+      </body></html>
+    `)
+})
+// localhost:3000/json
+
+app.get('/json', (req, res) => {
+  res.json(
+    {
+      technologies: [
+        'JavaScript', 'Noodle', 'Express']
+    })
+})
+
+// localhost:3000/admin/grades
+app.get('/admin/grades', (req, res) => {
+  res.sendStatus(401)
+})
+
+// localhost:3000/api/grades
+
+app.get('/api/grades', (req, res) => {
+  res.status(401).send('Crepper! Oh man!')
+})
+
+// localhost:3000/boredom
+app.get('/boredom', (req, res) => {
+  res.status(404).send('Sorry-you just be a friend!')
+})
+// localhost3000/html-pic
+app.get('/html-pic', (req, res) => {
+  res.send(`
+    <html>
+      <body>
+    <h1>hello boii</h1>
+    <img src="/cat.jpg" width="300" />
+  </body>
+  </html>
+  `)
+})
+
+// Start server
+const PORT = 3000
+app.listen(PORT, () => console.log(
+  `listening on http://localhost:${PORT}`))
+
+// Lab
+app.get('/Lab', (req, res) => {
+  res.send(`
+  <html><body>
+  <li>1. <a href="/profiles/Foon">/profiles/Foon</a></li>
+  <li>2. <a href="/api/profiles/Foon">/api/profiles/Foon</a></li>
+  <li>3. <a href="/cat-simulator">/cat-simulator</a></li>
+  <li>4. <a href="/admin/grades">/admin/grades</a> AND <a href="/api/grades">/api/grades</a> AND <a href="/boredom">/boredom</a></li>
```

Comment:
ฉันคิดว่าคุณยังไม่จบภาค 4 (รหัสสถานะ)
I think you haven't finished part 4 (status codes).

Reply:
![image](https://user-images.githubusercontent.com/84704807/123519940-5c045580-d6d8-11eb-86f5-7e80d17d50de.png)

I think this one is done.
![image](https://user-images.githubusercontent.com/84704807/123519962-7f2f0500-d6d8-11eb-941d-b2cd13e3d36e.png)
![image](https://user-images.githubusercontent.com/84704807/123519970-8eae4e00-d6d8-11eb-9575-9c894fd6f8ac.png)

Reply:
Ahh sorry; it was above `// Lab`.